layouts/index.redirects

```diff
@@ -1,3 +1,5 @@
 {{ $latest := site.Params.versions.latest }}
 /docs/latest     /docs/{{ $latest }}
 /docs/latest/*     /docs/{{ $latest }}/:splat
+/docs/v2    /docs/v2.3
```

Comment:
The rules are working. One last change: add a trailing slash to the end of this rule.

Reply:
Oh, and to the rule in line 2 as well :).